runtime/src/main/java/org/corfudb/runtime/clients/LayoutHandler.java

```diff
@@ -94,4 +105,116 @@ private static Object handleAlreadyBootstrap(CorfuMsg msg,
             throws Exception {
         throw new AlreadyBootstrappedException();
     }
+
+    // Protobuf region
+
+    /**
```

Comment:
Remove all usages of `CorfuMsgType` with these changes those paths become inactive. Its better to remove them than assume that they are not functional.